x/ibc/handler.go

```diff
@@ -37,22 +38,75 @@ func NewHandler(k Keeper) sdk.Handler {
 
 		// IBC channel msgs
 		case channel.MsgChannelOpenInit:
-			res, cap, err := channel.HandleMsgChannelOpenInit(ctx, k.ChannelKeeper, msg)
+			cbs := k.PortKeeper.LookupModule(ctx, msg.PortID)
+			portCap, _ := cbs.GetCapability(ctx, ibctypes.PortPath(msg.PortID))
```

Comment:
Get port capability from module using callback

Reply:
Why do we need to look this up? The module should already have the port capability.